deps/build.jl

```diff
@@ -50,4 +50,4 @@ provides(SimpleBuild,
              end)
           end), [libmumps_simple], os = :Unix)
 
-@BinDeps.install [:libmumps_simple => :libmumps_simple]
+@BinDeps.install Dict(:libmumps_simple => :libmumps_simple)
```

Comment:
Thanks for the hint. I can add that later. Shall I update the package requirement then, too? We should include https://github.com/JuliaLang/Compat.jl